helm/dns-operator-aws/templates/deployment.yaml

```diff
@@ -41,13 +41,14 @@ spec:
         - --management-cluster-namespace={{ .Values.managementClusterNamespace }}
         - --management-cluster-name={{ .Values.managementClusterName  }}
         - --workload-cluster-basedomain={{ .Values.workloadClusterBaseDomain }}
+        - --associate-resolver-rules={{ .Values.associateResolveRules }}
         resources:
           requests:
             cpu: 100m
-            memory: 50Mi
+            memory: 100Mi
           limits:
             cpu: 100m
-            memory: 80Mi
+            memory: 150Mi
```

Comment:
Not needed for this PR but do we maybe need a VPA for this app?

Reply:
yeah make sense, will drop a note

Reply:
there is actually a VPA, i guess I don't want this to get killed every time it want to boot :p